connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -82,18 +82,14 @@ def __get_cosign_validated_digests(self, image: str, pubkey: str):
                 except json.JSONDecodeError:
                     logging.info("non-json signature data from Cosign: %s", sig)
                     pass
-        elif "error: no matching signatures:\nfailed to verify signature\n" in stderr:
+        elif "Error: no matching signatures:\nfailed to verify signature\n" in stderr:
             msg = "Failed to verify signature of trust data."
             raise ValidationError(
                 message=msg,
                 trust_data_type="dev.cosignproject.cosign/signature",
                 stderr=stderr,
             )
-        elif re.match(
-            r"^error: fetching signatures: getting signature manifest: "
-            r"GET https://[^ ]+ MANIFEST_UNKNOWN:.*",
-            stderr,
-        ):
+        elif "Error: no matching signatures:\n\nmain.go:48" in stderr:
```

Comment:
Using the line of error thrown seems very brittle. Is this intentional to find changes or can the line be removed to cause less work for us?

Reply:
I would not see how we could distinguish between the two cases of no signature and wrong signature. Technically, would not need to have the line. Maybe I remove the exact line. That good for you?

Reply:
done!

Reply:
Yeah, that works. Didn't really grasp that this was for distinguishing. Then main is actually needed if they changed the above error method. Feels kinda shitty, but I guess it's AGAIG unless there's different exit codes for cosign